README.md

```diff
@@ -85,6 +85,13 @@ object_client.metadata.dublin_core
 
 # Get the public descriptive XML representation
 object_client.metadata.descriptive
+
+# Return the Cocina metadata
+object_client.find
```

Comment:
I don't love this name. Open to alternatives? Maybe `#get`?

Reply:
I'm trying to stick close to valkyrie: https://github.com/samvera/valkyrie/blob/master/lib/valkyrie/persistence/memory/query_service.rb#L18-L28

Reply:
ok, sold.